Automattic/wp-calypso
# Paid Stats: apply USD $5 as threshold for priority support

Related to #81531 

## Proposed Changes

* Changed starting value to USD $5 or equivalent
  * As a result of the above, the heart emoji and and priority support are aligned
* Only show "Priority Support" as benefit for PWYW >= $5 USD or equivalent 

P2: p1HpG7-orS-p2#comment-65838

## Testing Instructions

* Open Calypso Live Branch for a site without any Stats products `/stats/purchase/:siteSlug`
* Ensure default price is $5 USD or equivalent
* Move the price down
* Ensure "Priority Support" becomes unavailable
* Move price back up to >= $5 or equivalent
* Ensure "Priority Support" becomes available
* Ensure heart emoji on the scrollable flips to heart at $5 or equivalent too
* Ensure the old and new PWYW both work

<img width="679" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1425433/7df2bd3a-a04f-4af3-ae03-700a461c2822">


<img width="646" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1425433/2cd78fd9-d1cc-40ab-b455-2f345004a7ba">

<img width="642" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1425433/f62ed6ea-221f-40f9-ae10-60182020caeb">



## Pre-merge Checklist
